src/AutoMapper/AdvancedConfiguration.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace AutoMapper
+{
+    public class AdvancedConfiguration
+    {
+        /// <summary>
+        /// Action called against the IConfigurationProvider before it gets sealed
+        /// </summary>
+        public Action<IConfigurationProvider> BeforeSeal { get; set; }
```

Comment:
Yea I thought about that.  But I got the thing to work with AM.Collections with a single action, so haven't changed it yet.